### PR TITLE
Create Copy email action if the Email Message composer is not available

### LINKED ISSUE #1442: Customer Email not able to be copied
**Is your feature request related to a problem? Please describe.**

When you open an order, then click `View Billing Information` and try to select the email, if the default mail app isn't setup for the phone, the email field does nothing: 

https://cld.wthms.co/onp0m3
Image link: https://cld.wthms.co/onp0m3

**Describe the solution you'd like**

If the default mail app isn't setup, allow the email to be able to be copied so it can be pasted into another email app, instead of having that field do nothing.

**Describe alternatives you've considered**

Not sure what an alternative would be here. Allowing that field to be able to be copied seems like the best fit.

**Additional context**

These issues are semi-related, but not the same: 

#334 
#1124

### PR DESCRIPTION
**Pull request by mindgraffiti**
Closes #1442 

Allow the user to copy the email if the email message composer is not available.

**Notes:** I chose to display an action sheet because that's how the other actions are presented for the email & phone buttons on this screen.

### To test
0. Ensure the testing Order has a shipping email address.
1. Open in a simulator or have a device with no Mail accounts set up.
2. Navigate to Order > Order Details > View Billing Details and tap on the Contact Details email address cell.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift [hunk 1]
```diff
@@ -161,6 +161,31 @@ private extension BillingInformationViewController {
...
+
+    /// Create an action sheet that offers the option to copy the email address
+    ///
+    func displayEmailCopyAlert(from: UIViewController) {
+        guard let sourceView = from.view, let email = order.billingAddress?.email else {
+            return
+        }
+
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .primary
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift]: could you update the color to `.text` from the Dark mode color updates on `release/3.2`?

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift [hunk 2]
```diff
@@ -161,6 +161,31 @@ private extension BillingInformationViewController {
...
+            self?.sendToPasteboard(self?.order.billingAddress?.email)
+        }
+
+        let popoverController = actionSheet.popoverPresentationController
+        popoverController?.sourceView = sourceView
+        popoverController?.sourceRect = sourceView.bounds
+
+        from.present(actionSheet, animated: true)
+
+        ServiceLocator.analytics.track(.orderDetailCustomerPhoneMenuTapped)
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift]: probably the email menu tapped event?

Comment (mindgraffiti) [reply WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift]: 🤦‍♀ yes, the email menu tapped event. Thank you!

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift (hunk 1 above)
Comment (mindgraffiti) [reply WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift]: Normally I would use that color, except this is an action sheet button. I used [this Figma design](https://www.figma.com/file/S62xbAwtgoMgax4GcuFjKS/Dark%2FLight-modes?node-id=170%3A5076) as a reference.

**Comment by mindgraffiti** [comment]
Thanks @jaclync! Updates I made before merging this PR:

✅ changed action sheet text to use .text instead of .primary
✅ updated code so it's using the correct Tracks event
✅ remove newline after email address when copying the text
✅ updated screenshot in summary to reflect action sheet button color changes

**Pull request merged**